src/common/Navbar/UploadButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { FiUpload } from "react-icons/fi";
+
+//not sure about this part
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  onClick: any; //??
```

Comment:
onClick is a function that may or may return nothing so you can type it this way:
```
onClick: () => void; // note that if you dont expect to use the event provided by the click you dont need to put in the interface definition
```